### ISSUE TITLE
Windows wheels don't build using AppVeyor

### ISSUE THREAD

**Issue by MarkWieczorek** [SHTOOLS/SHTOOLS#223]
With the latest release of SHTOOLS 4.6.2, the appveyor windows builds are once again failing. I have no idea how to even debug this one....

https://ci.appveyor.com/project/MarkWieczorek/build-shtools/builds/31263348

**Comment by MarkWieczorek** [comment]
@alkalinin Would you be interested in taking a quick look at this new problem?

**Comment by alkalinin** [comment]
Yes, I will take a look in the coming days.

**Comment by alkalinin** [comment]
I have found the reason. In the current build environment the numpy package change the version from 1.17.4 to the 1.18.1. In the version 1.18.1 we do not need to patch the distutils.

In my fork I have commented the numpy patch, see https://github.com/alkalinin/build-shtools/commit/86750749cb64a3bbcbc1ea54d318ab0bf205bc9b and it was successfully built, see the [log](https://ci.appveyor.com/project/AlexanderKalinin/build-shtools/builds/31288987/job/cq5r76844hj6bah9).
